public/js/services/game.js

```diff
@@ -23,18 +22,18 @@ angular.module('mean.system')
       joinOverride: false
     };
 
-    var notificationQueue = [];
-    var timeout = false;
-    var self = this;
-    var joinOverrideTimeout = 0;
+    let notificationQueue = [];
+    let timeout = false;
+    let self = this;
+    let joinOverrideTimeout = 0;
 
-    var addToNotificationQueue = function(msg) {
+    let addToNotificationQueue = function (msg) {
       notificationQueue.push(msg);
       if (!timeout) { // Start a cycle if there isn't one
         setNotification();
       }
     };
-    var setNotification = function() {
+    var setNotification = function () {
```

Comment:
When using ES6 then use it in the whole file for uniformity. Therefore, this should be `let` not `var`